pyogrio/tests/test_geopandas_io.py

```diff
@@ -1488,11 +1557,17 @@ def test_write_geometry_z_types_auto(
     assert_geodataframe_equal(gdf, result_gdf)
 
 
-def test_read_multisurface(data_dir):
-    df = read_dataframe(data_dir / "test_multisurface.gpkg")
+def test_read_multisurface(data_dir, use_arrow):
+    if use_arrow:
+        with pytest.raises(shapely.errors.GEOSException):
```

Comment:
In this case I kept it in a single test because it's a very short one (that keeps it IMO a bit clearer what the difference between with and without arrow is, in this simple case)